operators/src/util/gdal.rs

```diff
@@ -73,6 +73,17 @@ pub fn add_ndvi_dataset(ctx: &mut MockExecutionContext) -> DatasetId {
     id
 }
 
+// TODO: move test helper somewhere else?
```

Comment:
Maybe put into gdal util in datatypes and remove TODO

Reply:
0d4e7c1

Reply:
Should I move the other test helpers there to?